controllers/funding/programmes.js

```diff
@@ -128,27 +136,114 @@ function initProgrammesList(router, config) {
+                    throw new Error('NoContent');
+                }
             })
+            .catch(err => {
+                renderNotFoundWithError(err, req, res);
+            });
+    };
+}
+
+function initProgrammeDetail(router) {
+    router.get('/programmes/:slug', (req, res) => {
+        handleProgrammeDetail(req.params.slug)(req, res);
+    });
+}
+
+function initProgrammeDetailAfaEngland(router, options) {
+    const testFn = ab.test('blf-afa-rollout-england', {
+        cookie: {
+            name: config.get('cookies.abTestAwardsForAll'),
+            maxAge: moment.duration(4, 'weeks').asMilliseconds()
+        },
+        id: 'O9FsbkKfSeOammAP0JrEyA'
+    });
+
+    // const percentageForTest = config.get('abTests.tests.awardsForAll.percentage');
+    const percentages = splitPercentages(50);
+
+    const getSlug = urlPath => last(urlPath.split('/'));
+
+    const replacementText = {
+        en: stripIndent`
+            <p>There are currently two ways to apply: via PDF application form or via online application form.</p>
+            <h2>Applying using the online form</h2>
+            <p>We are trialling a new application process where you can submit an application via a web form, rather than downloading a PDF.</p>
+            <p><a href="https://apply.biglotteryfund.org.uk/?cn=en" class="btn">Apply Online</a></p>
+            <p>This new form is a trial, and only open to a limited number of applicants. We’d love your feedback on how you find the experience.</p>
+            <h2><strong>Applying via email or post</strong></h2>
+            <p>You are also welcome to apply via email or post using our PDF form. To apply this way, follow the instructions below.</p>
+            <ol><li><p>Please <a href="https://www.biglotteryfund.org.uk/-/media/Files/Programme%20Documents/Awards%20for%20All%20England/NLA4A_guide_england.pdf">read this guidance document first</a> (PDF 650KB).</p></li><li><p><a href="https://www.biglotteryfund.org.uk/-/media/Files/Programme%20Documents/Awards%20for%20All%20England/NLA4A_app_england.pdf">Save the application form</a> (PDF 700KB) to your computer and open it with the latest version of Adobe Reader. Do not use Preview or any other application. <strong>Please note we are no longer able to accept older versions of the application form.</strong></p></li><li><p>The easiest way is to fill in the form within the file itself – you don’t need to print it off, unless you prefer to send it by post.</p></li><li><p>Email or post the completed application form to us.</p></li></ol><p>We’ll let you know our decision in about ten weeks.</p>
+            <p>If you need more information, please contact us at <a href="mailto:general.enquiries@awardsforall.org.uk">general.enquiries@awardsforall.org.uk</a> or call our advice team on <strong>0345 4 10 20 30</strong>. Text Relay users, please use <strong>18001</strong> plus <strong>0345 4 10 20 30</strong>.</p>
+        `,
+        // @TODO: Add translation
```

Comment:
Update this once the new page is approved and we have revised Welsh language copy.